.ci/build.sh

```diff
@@ -14,9 +19,34 @@ else
   python manage.py fetch_deployed_data _site $EXPORTED_DATA
 fi
 
+# fetch deployed meta_review data
+python manage.py fetch_deployed_data --allow_failure \
```

Comment:
You could add `static/meta_review.json` to `EXPORTED_DATA` , and *always* run `python manage.py fetch_deployed_data --allow_failure _site $EXPORTED_DATA`  *before* the `GCI_TOKEN` token command above.

(We didnt have `--allow_failure` / `--allow-failure` , so it needed the `if` )